cmd/cluster/analyze/analyze.go

```diff
@@ -40,6 +40,10 @@ const (
 	flagVerbose      = "verbose"
 	flagVerboseShort = "v"
 	flagVerboseHelp  = "Display additional detailed information related to the analysis."
+
+	flagJSON      = "json"
+	flagJSONShort = "j"
```

Comment:
Can we do without this shortcut?  I am not sure we need to have one for every option.

Reply:
Ok! I removed the shortcut flag for this one.